demo/src/main/java/com/paypal/messagesdemo/XmlActivity.kt

```diff
@@ -29,23 +30,181 @@ class XmlActivity : AppCompatActivity() {
 
 		val payPalMessage = binding.payPalMessage
 		val progressBar = binding.progressBar
-		val reloadButton = binding.reloadButton
 
-		// TODO add example of adding MessageView here instead of in XML
+		val editedClientId: EditText? = findViewById<EditText>(R.id.clientId)
+
+		val logoTypeRadioGroup = findViewById<RadioGroup>(R.id.logoTypeRadioGroup)
+		logoTypeRadioGroup.setOnCheckedChangeListener { _, checkedId ->
+			when (checkedId) {
+				R.id.stylePrimary -> {
+					logoType = PayPalMessageLogoType.PRIMARY
+				}
+				R.id.styleInline -> {
+					logoType = PayPalMessageLogoType.INLINE
+				}
+				R.id.styleAlternative -> {
+					logoType = PayPalMessageLogoType.ALTERNATIVE
+				}
+				R.id.styleNone -> {
+					logoType = PayPalMessageLogoType.NONE
+				}
+			}
+		}
```

Comment:
The logic for all these listeners is solid, but we could make the code simpler by setting it to an expression

```
			logoType = when (checkedId) {
				R.id.stylePrimary -> PayPalMessageLogoType.PRIMARY
				R.id.styleInline -> PayPalMessageLogoType.INLINE
				R.id.styleAlternative -> PayPalMessageLogoType.ALTERNATIVE
				R.id.styleNone -> PayPalMessageLogoType.NONE
				else -> PayPalMessageLogoType.PRIMARY
			}
```